.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "schema-registry"]
```

Comment:
I'd prefer if we can avoid git submodules :X

maybe we can do something with go modules? like we auto-generate a go file per avro schema that we commit to the `schema-registry` repo that contains an embedded copy of the schema. then setup CI so we don't forget to update those generated files?

maybe best as a follow up item - maybe we just drop a standalone copy of `suggestion.avsc` for now?

Reply:
SO GLAD you have another idea for that haha :) Did the submodule since that is typical but I had it too.  I'll drop a static file for now.